spec/models/group_spec.rb

```diff
@@ -75,4 +75,23 @@
       end
     end
   end
+
+  describe "group_invite_statuses", :vcr do
+    let(:organization) { classroom_org }
+    let(:grouping)     { create(:grouping, organization: organization) }
+    let(:group)        { Group.create(grouping: grouping, title: "#{Faker::Company.name} Team") }
+    let(:invitation)   { create(:group_assignment_invitation) }
+
+    it "returns a list of invite statuses" do
+      group_invite_status = GroupInviteStatus.create(group: group, group_assignment_invitation: invitation)
+      expect(group.group_invite_statuses).to eq([group_invite_status])
+    end
+
+    it "on #destroy destroys invite status and not the invitation" do
```

Comment:
Does it make sense to add a test here that makes sure that all the `GroupInviteStatuses` of a group get destroyed when the group is destroyed?

Reply:
Yea agreed we would just be testing native functionality, I was thinking it might be nice to document the `dependent: :destroy` relationship just in case of future breaking changes?